message_definitions/v1.0/messages.xml

```diff
@@ -2477,19 +2477,20 @@
   <message name="GUIDED_SETPOINT_NED" id="40" link="forwarded">
     <description>
       Set vehicle position or velocity in NED.
-      Frame can be specified with the bits 0-3 in the flags field:
-      0x0: LOCAL_NED
-      0x1: LOCAL_OFFSET_NED
-      0x2: BODY_NED
-      0x3: BODY_OFFSET_NED
-      Bits 4-7 in the flag field can be used to enable velocity control on specifc fields:
-      bit 4: x as velocity
-      bit 5: y as velocity
-      bit 6: z as velocity
+      Frame can be specified with the bits 0-3
+      Velocity of position setpoint can be specified with the bits 5-7
+      Flags field definition:
+      bit 0: x,y as offset coordinates
```

Comment:
I think it would be nice to use a list for the bits, so it's nicely displayed in the generated docs: http://docs.paparazziuav.org/latest//paparazzi_messages.html#GUIDED_SETPOINT_NED

so instead:
```
- bit 0: foo
- bit 1: bar
```